content/guides/component-testing/mounting-react.md

```diff
@@ -14,105 +14,103 @@ end-to-end test.
-will have registered the `mount` method as a Cypress Command, and it will be
-available in any Cypress test under `cy.mount`.
+</code-block>
+</code-group>
 
-If you're using providers or other global app-level setups in your React app,
-you'll want to follow the
-"[Customizing cy.mount guide for React](/api/commands/mount)".
+This allows you to use `cy.mount` in any component test without having to import
+the framework-specific mount command.
 
-<!-- TODO: link to customizing cy.mount command -->
+You can customize `cy.mount` to fit your needs. For instance, if you are using
+providers or other global app-level setups in your React app, you can configure
+them here. For more info, see the
+[Customizing cy.mount guide for React](/guides/component-testing/custom-mount-react).
 
-```js
-// cypress/support/component.js
-import { mount } from 'cypress/react'
+### Your First Component Test
 
-Cypress.Commands.add('mount', mount)
-```
+Now that you have a component and know how to mount it let's write our first
+test.
 
-Now we can update our `<Stepper />` component by removing the `mount` import and
-using the `cy.mount` command. Go ahead and create a spec in your project named
-`Stepper.spec.jsx` in the same directory as `Stepper.jsx` and copy the following
-into it:
+To get started, create a spec file in the same directory as the `Stepper.jsx`
+component and name it **Stepper.cy.jsx**. Then paste the following into it:
 
 <code-group>
 <code-block label="Stepper.cy.jsx" active>
 
-```jsx
+```js
 import Stepper from './Stepper'
 
 describe('<Stepper>', () => {
   it('mounts', () => {
-    cy.mount(<Stepper />) // this command now works in any test!
+    cy.mount(<Stepper />)
   })
 })
 ```
 
 </code-block>
 </code-group>
 
+Here, we have a single test that ensures that our component mounts.
+
 ## Running the Test
 
-Now it's time to see the test in action. Open up Cypress if not already:
+Now it's time to see the test in action. Open up Cypress if it is not already
+running:
 
 ```bash
-npx cypress open
+npx cypress open --component
 ```
 
+> The `--component` flag will launch us directly into component testing
+
 And launch the browser of your choice. In the spec list, click on
-**Stepper.cy.jsx** and you will see the stepper component mounted in the test
+**Stepper.cy.js** and you will see the stepper component mounted in the test
 area.
 
-Having a basic test that mounts a component in its default state is a good way
-to get started testing. Since Cypress renders your component in a real browser,
-you can visually see if it is rendering how it should, and you can also play
-around with it manually in the test runner.
+<DocsImage 
+  src="/img/guides/component-testing/first-test-run-react.png" 
+  caption="Stepper Mount Test"> </DocsImage>
 
-<!-- TODO - picture of mounted stepper component -->
+A basic test that mounts a component in its default state is an excellent way to
```

Comment:
This could be a partial since it's used on a couple of different pages. Unless we want the subtle differences in verbiage.

Reply:
I was fine with the duplication at this point as we might be able to do some more advanced things in the new docs site